tc/autotuner/genetic_search.cc

```diff
@@ -166,6 +195,7 @@ GeneticSearch::GeneticSearch(
       lastBestConf(confs[0]),
       numGenerations(numGenerations),
       maxPopulationSize(populationSize),
+      matingPoolSize(populationSize * 3),
```

Comment:
In one of the subsequent commits I introduced a separate flag for the mating pool size (instead of hardcoding `populationSize * 3`).